### ISSUE TITLE
Typing error in build for Angular 11

### ISSUE THREAD

**Issue by maxailloud** [Stabzs/Angular2-Toaster#221]
Hello.
I just updated my project to the last version of the package alongside Angular 11.
I am having an issue coming from the typing of it.

```
Error: node_modules/angular2-toaster/lib/toaster-config.d.ts:5:25 - error TS1110: Type expected.

5     [key in ToastType]: ;
                          ~
node_modules/angular2-toaster/lib/toaster-config.d.ts:8:25 - error TS1110: Type expected.

8     [key in ToastType]: ;
                          ~
```

It looks like the ToasterConfig class typings has some missing type for two different constants.
Here is the code the errors are from:

```
export declare const DefaultTypeClasses: {
    [key in ToastType]: ;
};
export declare const DefaultIconClasses: {
    [key in ToastType]: ;
};
```

No idea what went wrong but it looks like it went wrong for this.

**Comment by Stabzs** [comment]
Thanks for reporting this.  I'm looking into it.

**Comment by maxailloud** [comment]
No problem. Thank you.

**Comment by Stabzs** [comment]
Looks like the type for the value is missing.  Publishing a new version as 11.0.1 shortly.

**Comment by Stabzs** [comment]
Published as 11.0.1 and i'll be unpublishing 11.0.0.  Please let me know if things look correct.

**Comment by maxailloud** [comment]
Fixed to me.
Thank you for being reactive.
